src/libslic3r/GCode.cpp

```diff
@@ -4152,18 +4209,22 @@ LayerResult GCode::process_layer(
                 if (m_config.reduce_crossing_wall)
                     m_avoid_crossing_perimeters.init_layer(*m_layer);
 
+                std::string start_str;
+                std::string start_str_a;
+                std::string temp_start_str;
                 if (this->config().gcode_label_objects) {
-                    gcode += std::string("; printing object ") + instance_to_print.print_object.model_object()->name +
```

Comment:
Did you changed the formatting of the gcode debug information for the object sequence on purpose? I imagine it could be a bit cleaner, just wondering. Was not part of the BambuStudio PR, but why not.

Reply:
BS inserts the debug comments in reverse order compared to OS. BS uses temp_start_str as a condition for adding **end_str += "M625\n;"**, which was my intent. I had some things come up that prevented me from getting things finished. I can probably just strike those parts out, though the original functionality remains as it was before. Never enough time.

Reply:
Thanks @tsmith35, that makes sense. No worries, you focused on the important changes and they look fine 👍🏻